minard/templates/shifter_information.html

```diff
@@ -0,0 +1,62 @@
+{% from "_formhelpers.html" import render_field_shifter %}
+{% extends "layout.html" %}
+{% block title %} Shifter Information  {% endblock %}
+{% block head %}
+  {{ super() }}
+{% endblock %}
+{% block body %}
+  {{ super() }}
+
+<div class="container">
+  <div class="row">
+    <h1 align="center">Shifter Information</h1> <br>
+    <table class="table table-hover">
+      <tr> 
+        <th> <h3> <font color='green'> {{ shifter }} </font> </h3> </th>
+        <th> <h3> <font color='green'> {{ updates }} </font> </h3> </th>
+        <th> <h3> <font color='green'> {{ expert }} </font> </h3> <br> </th>
+      </tr>
+    </table>
+    <h3> If you are the current shifter, please fill out the below information at the start of your shift. If you are not the current shifter, do not fill in this form. After hitting Submit, make sure the current shifter and expert information about update properly. If multiple people are on shift, the shift leader should fill this out. This information will only be used for email and text alarm alerts. </h3> <br>
+    <form action="shifter_information" method="POST" role="form">
+      <table class="table table-hover">
+         <tr> 
+           <th> <h4> First name: </h4> </th>
+           <th> <h4> {{ render_field_shifter(form.firstname) }} </h4> </th>
+           <th> Required </th>
+         </tr>
+         <tr>
+           <th> <h4> Last name: </h4> </th>
+           <th> <h4> {{ render_field_shifter(form.lastname) }} </h4> </th>
+           <th> Required </th>
+         </tr>
+         <tr>
+           <th> <h4> Expert: </h4> </th>
+           <th> <h4> {{ form.expert }} </h4> </th>
+           <th> Required, current on-call expert. </th> 
+         </tr>
+         <tr>
+           <th> <h4> Email: </h4> </th>
+           <th> <h4> {{ render_field_shifter(form.email) }} </h4> </th>
+           <th> Optional, leave empty to not receive email alerts.<br>
+                Use full address (eg, username@aol.com) </th>
+         </tr>
+         <tr>
+           <th> <h4> Phone number: </h4> </th>
+           <th> <h4> {{ render_field_shifter(form.phone) }} </h4> </th>
+           <th> Optional, leave empty to not receive phone alerts. <br> 
+                Don't use spaces and include the international code (eg, 15559998888) </th>
+         </tr>
+         <tr>
+           <th> <h4> Country: </th>
+           <th> <h4> {{ render_field_shifter(form.country) }} </h4> </th>
+           <th> Optional, leave empty to not receive phone alerts. <br>
+                This is the country associated with your phone number. <br> 
+                Current valid countries: USA, Canada, Portugal, Germany, UK </th>
+         </tr>
+      </table>
+      <h2> <button type="submit">Submit</button> </h3>
```

Comment:
another purely stylistic change that you're free to ignore, but I don't think this needs to be a header element. Also you've got h2 and h3 mixed.